samples/drivers/spi_flash_at45/Kconfig

```diff
@@ -0,0 +1,16 @@
+#  Copyright (c) 2020 Gerson Fernando Budke
+#  SPDX-License-Identifier: Apache-2.0
+
+menu "SPI Flash AT45 - Sample app Configuration"
+
+config SPI_FLASH_AT45_SAMPLE_DEVICE
```

Comment:
This isn't the right solution.  Instead the sample should use:

```
#define FLASH_DEVICE DT_LABEL(DT_INST(0, atmel_at45))
```
like the `spi_flash` example does and to guarantee that the sample will be using the name that's used by the driver.

Reply:
Agreed! However can have boards with more than on external memory.
I suggest introduce `ext-flashN` alias.

Reply:
Done!